src/molecules/Header/index.js

```diff
@@ -1,12 +1,12 @@
 import React from "react"
 import { Link } from "gatsby"
-import "./styles.css"
+import styles from "./header.module.css"
 
 const Header = ({ siteTitle }) => (
   <header>
-    <h1 className="logo">
```

Comment:
Because of outline, we need to use `<h1>` as website name.
For more context, see [Using `<h1>` on website name in HTML5](https://stackoverflow.com/questions/14912778/using-h1-on-website-name-in-html5)

Reply:
Done.